Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static int ConvertStringToInt(string message)
+        {
+            Console.WriteLine(message);
+            return Convert.ToInt32(Console.ReadLine());
+        }
+
+        public static string ConvertStringToString(string message)
+        {
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        public static float ConvertStringToFloat(string message)
+        {
+            Console.WriteLine(message);
+
+            return float.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
+
+        }
+        public static float Bmi(float weight, float heigth)
+        {
+            return weight / (float)Math.Pow(heigth, 2);
```

Comment:
As far as I have understood the homework those functions should be part of your Program.cs.
You definitly need to create a Demo() function here and fill in what is in the Main() function of Program.cs.

Reply:
At the the time I finished the homework this was not in the requirements.